src/app.js

```diff
@@ -6,8 +6,75 @@ import _ from 'underscore';
 import './css/foundation.css';
 import './css/style.css';
 
-console.log('it loaded!');
+//models and collections
+import Trip from './models/trip';
+import TripList from './collections/trip_list';
+
+
+const tripList = new TripList();
+
+let tripTemplate;
+let tripInfoTemplate;
+const render = function(tripList) {
+  console.log('in render');
+  $('#trip-list').append('<tr><th>ID</th><th>CONTINENT</th><th>COST</th><th>CATEGORY</th></tr>');
```

Comment:
Before we start filling in the contents of the trip list we should clear it out by calling `$('#trip-list').empty();`.

This allows the `render` function to be _idempotent_, so that if our application has to call it multiple times (such as when adding or removing a trip), it will still do the correct thing.